app/controllers/contestant_projects_controller.rb

```diff
@@ -0,0 +1,7 @@
+class ContestantProjectsController < ApplicationController
+  def create
+    project = Project.find(params[:id])
+    contestant_project = ContestantProject.create(contestant_id: params[:contestant_id], project_id: params[:id])
+    redirect_to "/projects/#{project.id}"
+  end
+end
```

Comment:
Good call doing this in the ContestantProjectsController. On line 5, I'd suggest using params[:id] insteaad of finding the project and using that. You already have everything you need coming through params :)